DropboxLike.Domain/Repositories/User/UserRepository.cs

```diff
@@ -16,15 +16,15 @@ public class UserRepository : IUserRepository
 {
     private readonly ApplicationDbContext _applicationDbContext;
     private readonly IAmazonS3 _awsS3Client;
-    private readonly string _bucketName;
+    private readonly string? _bucketName;
```

Comment:
This should not be nullable as it comes from the configuration that should be non-null.